### ISSUE TITLE
The Legend of Zelda: Breath of the Wild

### ISSUE THREAD

**Issue by HorrorTroll** [Ryujinx/Ryujinx-Games-List#86]
Current of `master`:
```
Unhandled Exception: System.NotImplementedException: nfp:user IUser: 2
   at Ryujinx.HLE.HOS.Services.IpcService.CallMethod(ServiceCtx Context) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Services\IpcService.cs:line 83
   at Ryujinx.HLE.HOS.Ipc.IpcHandler.IpcCall(Switch Ns, Process Process, AMemory Memory, KSession Session, IpcMessage Request, Int64 CmdPtr) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Ipc\IpcHandler.cs:line 33
   at Ryujinx.HLE.HOS.Kernel.SvcHandler.ProcessIpcRequest(Object State) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Kernel\SvcSystem.cs:line 231
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ThreadPoolWorkQueue.Dispatch()

**Comment by ReaperOfSouls1909** [comment]
![image](https://user-images.githubusercontent.com/18043258/76137197-58931980-6010-11ea-9a9b-cf92b4807c0d.png) 
![image](https://user-images.githubusercontent.com/18043258/76137207-7f515000-6010-11ea-829a-984ad6e3be9e.png)
 
![image](https://user-images.githubusercontent.com/18043258/76137212-87a98b00-6010-11ea-8198-fe6860d88999.png)

Graphics are Looking great depending on your luck  cause sometimes the gfx will be broken on boot but fps is between 5-17fps depending on the area

**Comment by nadimmoran** [comment]
Estado actual in Last Master:
![Captura de pantalla (789)](https://user-images.githubusercontent.com/61863609/76137217-3ac1b680-6008-11ea-8856-f9666fc257a7.png)
![Captura de pantalla (785)](https://user-images.githubusercontent.com/61863609/76137220-3c8b7a00-6008-11ea-9dfa-1c6d1c9a4766.png)
![Captura de pantalla (784)](https://user-images.githubusercontent.com/61863609/76137221-3d241080-6008-11ea-995f-448d4d707672.png)
![Captura de pantalla (777)](https://user-images.githubusercontent.com/61863609/76137222-3eedd400-6008-11ea-8028-718b7c3df52b.png)
![Captura de pantalla (771)](https://user-images.githubusercontent.com/61863609/76137224-40b79780-6008-11ea-9e3f-02a741927b5b.png)

**Comment by ReaperOfSouls1909** [comment]
![image](https://user-images.githubusercontent.com/18043258/76137230-b4f63900-6010-11ea-94ab-7f4a610febb4.png)
major issues sometimes the gfx will boot broken as seen in this image

**Comment by B3astTech** [comment]
Updated

**Comment by Ghost6713** [comment]
how i dowload this game

**Comment by ReaperOfSouls1909** [comment]
newest master 
[Ryujinx_2020-04-30_12-03-42.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4559161/Ryujinx_2020-04-30_12-03-42.zip)
update 1.5.0 stops at map 257

**Comment by braddevans** [comment]
```
00:07:25.320 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007190 (0x00000000000017E0).
00:07:25.320 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007198 (0x000000000A201260).
00:07:25.327 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007790 (0x0000000000000000).
00:07:25.329 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007F10 (0x00000002ACD9E660).
00:07:25.329 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007F18 (0x00000000AB685E26).
00:07:25.329 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x007F28 (0x00000002ACD9E6B8).
00:07:25.329 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x008518 (0x00000002ACD9EF40).
00:07:25.332 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x008DC8 (0x00000002ACD9F698).
00:07:25.332 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x008DD0 (0x0000000000000000).
00:07:25.335 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x0096D8 (0x00000002ACD9FE60).
00:07:25.335 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x0096E8 (0x00000002ACD849D8).
00:07:25.335 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x0096F0 (0x0000000000000000).
00:07:25.335 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x0096F8 (0xFFFFFFFF0000FFFF).
00:07:25.335 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x009708 (0x0000000000000001).
00:07:25.338 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00A3F8 (0x42DBA6C000000000).
00:07:25.338 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00A408 (0x44D642A03F800000).
00:07:25.338 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00A410 (0x0000000000000000).
00:07:25.338 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00A418 (0x0000000000000000).
00:07:25.339 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00AA58 (0x0000000000001800).
00:07:25.339 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00B070 (0x00000002ACE05C60).
00:07:25.340 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00BD90 (0x00000002ACDA2540).
00:07:25.343 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00C638 (0x00000002ACDA364C).
00:07:25.343 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00C648 (0x00000002ACDA36DC).
00:07:25.344 | GUI.RenderLoop Gpu Log: Shader translator: Invalid instruction at 0x00D618 (0x000000000A1DB788).
```

**Comment by Doctorwho1909** [comment]
[Ryujinx_2020-10-11_01-53-56.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/5360495/Ryujinx_2020-10-11_01-53-56.log)
1.6.0 update needs  Ryujinx.HLE.Exceptions.ServiceNotImplementedException: Ryujinx.HLE.HOS.Services.Am.AppletAE.AllSystemAppletProxiesService.SystemAppletProxy.ICommonStateGetter: 54

**Comment by jwpjrdev** [comment]
Hope to see more performance updates on this. Would be great to play on mac.

**Comment by dipeve** [comment]
While running the game, the Camera Rune doesn't identify anything when pointing it, it shows an orange question mark.
Otherwise, the game runs relatively well (19-30 fps), v1.0.6781 on a i5 10400, with some random crashes from time to time.


![botw camera rune](https://user-images.githubusercontent.com/53589762/111017447-6e10f100-8392-11eb-85e9-2dc4d736735c.PNG)

**Comment by HayoPepper** [comment]
I was in the middle of a battle with a starter orc [I forgot the name but I'm new to the game sorry], anyway it crashed and I thought sharing the crash log might be helpful 

[Ryujinx_1.0.6797_2021-03-23_17-22-01.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/6188635/Ryujinx_1.0.6797_2021-03-23_17-22-01.log)

**Comment by LukeWarnut** [comment]
## The Legend of Zelda: Breath of the Wild

#### Game Update Version : 1.6.0

#### Current on `master` : 1.0.6945

Playable overall, but with multiple issues:
- The game struggles to reach 30FPS, but doesn't hinder the playability too much, as the game's speed does not depend on the frame rate.
- Upon starting a new file, the Ancient Shaft item did not appear correctly in menus. This error did not occur when using the save file from my Switch.
- When teleporting, Link doesn't appear during the animation, only the blue effects. Link only appears again after the animation is complete. A similar effect appears to take place when enemies are defeated.
- I encountered a large vertex explosion at the Woodland Stable from the kid that grants the Octo Balloon side quest.
- The game crashed during gameplay a single time.

```
01:56:08.476 |E| GUI.RenderLoop Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Not mapped: va=0xFFFFFFFFFFFFFFFF, size=0x0000000000000004
   at Ryujinx.Cpu.MemoryManagerHostMapped.AssertMapped(UInt64 va, UInt64 size)
   at Ryujinx.Cpu.MemoryManagerHostMapped.GetSpan(UInt64 va, Int32 size, Boolean tracked)
   at Ryujinx.Graphics.Gpu.Memory.MemoryManager.ReadImpl(UInt64 va, Span`1 data, Boolean tracked)
   at Ryujinx.Graphics.Gpu.Memory.MemoryManager.Read[T](UInt64 va)
   at Ryujinx.Graphics.Shader.Translation.ShaderHeader..ctor(IGpuAccessor gpuAccessor, UInt64 address)
   at Ryujinx.Graphics.Shader.Translation.Translator.DecodeShader(UInt64 address, IGpuAccessor gpuAccessor, TranslationFlags flags, TranslationCounts counts, ShaderConfig& config)
   at Ryujinx.Graphics.Shader.Translation.Translator.CreateContext(UInt64 address, IGpuAccessor gpuAccessor, TranslationFlags flags, TranslationCounts counts)
   at Ryujinx.Graphics.Gpu.Shader.ShaderCache.GetGraphicsShader(GpuState state, ShaderAddresses addresses)
   at Ryujinx.Graphics.Gpu.Engine.Methods.UpdateShaderState(GpuState state)
   at Ryujinx.Graphics.Gpu.Engine.Methods.UpdateState(GpuState state, Int32 firstIndex, Int32 indexCount)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 firstIndex, Int32 indexCount)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 argument)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Send(MethodParams meth)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Process(ReadOnlySpan`1 commandBuffer)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoDevice.DispatchCalls()
   at Ryujinx.Ui.RendererWidgetBase.Render()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

#### Hardware Specs : 

##### CPU: AMD Ryzen 5 2600
##### GPU: NVIDIA GTX 1660 SUPER
##### RAM: 16GB

#### Screenshots :

![image](https://user-images.githubusercontent.com/65521430/124207580-d1df3700-dab3-11eb-8682-6e3d8421bb60.png)
![image](https://user-images.githubusercontent.com/65521430/124207588-d4419100-dab3-11eb-9884-916a4088d8bc.png)
![image](https://user-images.githubusercontent.com/65521430/124207597-d7d51800-dab3-11eb-8975-c5d22e572b44.png)
![image](https://user-images.githubusercontent.com/65521430/124207665-00f5a880-dab4-11eb-8c72-be86b77a495e.png)
![image](https://user-images.githubusercontent.com/65521430/124207609-dd326280-dab3-11eb-878c-56c1bbe10a15.png)
![image](https://user-images.githubusercontent.com/65521430/124207687-0b17a700-dab4-11eb-8084-4894d13375da.png)
![image](https://user-images.githubusercontent.com/65521430/124207698-1074f180-dab4-11eb-93b6-8a4d861bdaf2.png)
![image](https://user-images.githubusercontent.com/65521430/124207703-15d23c00-dab4-11eb-9af1-a3c3b4a41cd0.png)
![image](https://user-images.githubusercontent.com/65521430/124207709-1a96f000-dab4-11eb-8585-3561b18f0dae.png)
![image](https://user-images.githubusercontent.com/65521430/124207720-1ff43a80-dab4-11eb-8b46-f8731e0a16e0.png)
![image](https://user-images.githubusercontent.com/65521430/124207727-2387c180-dab4-11eb-8564-cbbf08f870d9.png)
![image](https://user-images.githubusercontent.com/65521430/124207728-25ea1b80-dab4-11eb-9a55-0582298a5a9c.png)
![image](https://user-images.githubusercontent.com/65521430/124207732-2a163900-dab4-11eb-8de7-8247abaa0466.png)
![image](https://user-images.githubusercontent.com/65521430/124207738-2c789300-dab4-11eb-817c-d7dcb961870d.png)
![image](https://user-images.githubusercontent.com/65521430/124207743-2e425680-dab4-11eb-94bc-406fa1a62086.png)


#### Log file :

[BOTW1Crash.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/6751534/BOTW1Crash.log)
[BOTW2.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/6751546/BOTW2.log)
[BOTW3.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/6751549/BOTW3.log)

**Comment by MutantAura** [comment]
### The Legend of Zelda: Breath of the Wild
Game Update Version : 1.6.0

Current on master : 1.0.6957
Barely playable even on high end hardware.

Specs
CPU: R5 5600X
GPU: RTX 3070 Founders
RAM: 32GB 3600Mhz DDR4
OS: Windows 10.0.19042 Build 19042
 

- Vsync issue where the game artificially locks itself to 20fps, the static 30fps mod is needed to overcome this as of this build.
![image](https://user-images.githubusercontent.com/44103205/125169490-5875d180-e1a2-11eb-87e0-45385ef4765d.png)
![image](https://user-images.githubusercontent.com/44103205/125169518-7c391780-e1a2-11eb-8dd8-adc021735730.png)


- Game struggles to reach 30fps even with Vsync off. On my hardware I'm regularly capping out at _barely_ above 30fps at around 37-38fps once area has stabilised (more on this later). On weaker hardware this could be a serious threat to even achieving native framerate.
![image](https://user-images.githubusercontent.com/44103205/125169363-b48c2600-e1a1-11eb-9900-071df4a3542b.png)
![image](https://user-images.githubusercontent.com/44103205/125169403-e604f180-e1a1-11eb-973a-b82418ca2c87.png)
_best performance seen at coastlines_
![image](https://user-images.githubusercontent.com/44103205/125169462-295f6000-e1a2-11eb-9d79-d11a2a1a0de8.png)

- Teleport bug as reported many a time where link is not rendered correctly
![image](https://user-images.githubusercontent.com/44103205/125169417-fb7a1b80-e1a1-11eb-8504-391528f00042.png)

- Various graphical bugs. Random items when picked up will have garbled textures. Don't have screenshots as I'm unable to recreate them. Ancient gears and springs were the main two I saw when first collecting.

- **STUTTER**
Shader stutter aside the game in its current state is unstable in framerate. Even the same area from the same save file with all shaders cached etc _will_ stutter completely randomly. This is the main issue for me that pushes it into the "unplayable" category. Something is wrong with this game currently and I have a feeling the stutters and performance let-downs are linked.

**Comment by MutantAura** [comment]
Some detail to the above post.

5 ish min video here (static 30fps mod enabled) showcasing some of the random stutters, teleport weirdness etc. 
https://www.youtube.com/watch?v=miB9xo29s0E

Something I forgot to add is that crashes are quite frequent at least on my system and this master version. See a log here. This crash log is consistent each time the game crashes. 
[Ryujinx_1.0.6966_2021-07-14_13-04-53.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/6816072/Ryujinx_1.0.6966_2021-07-14_13-04-53.log)

**Comment by Niwu34** [comment]
Graphic Bug when using the Bike which can be unlocked in the DLC of the game
![green](https://user-images.githubusercontent.com/67392333/149214120-029127fd-0486-46f5-9b35-88f7c216c324.png)
![unknown](https://user-images.githubusercontent.com/67392333/149214133-924d4ef5-0aab-4ff0-baa4-f9bb80d16923.png)

Can be temporary fixed by teleporting (see Vide Below)
https://cdn.discordapp.com/attachments/814263228926132274/927660442829082744/2022-01-03_21-26-21_Trim.mp4

**Comment by joseph0817** [comment]
bro how do i download it

**Comment by nonchip** [comment]
crashed when killing one of those red goblin people right after talking to the hooded guy at the start of the game:
```
$ ./Ryujinx            
00:00:00.004 |I| Configuration LogValueChange: EnableFileLog set to: True
00:00:00.135 |I| Configuration LogValueChange: BackendThreading set to: On
00:00:00.138 |I| Configuration LogValueChange: ResScale set to: 2
00:00:00.147 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.148 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.151 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x10
00:00:00.172 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.104
00:00:00.200 |N| Application Print: Operating System: Linux 5.15.32_1 #1 SMP Mon Mar 28 15:46:09 UTC 2022 (X64)
00:00:00.200 |N| Application Print: CPU: AMD Ryzen 5 2600 Six-Core Processor ; 12 logical
00:00:00.202 |N| Application Print: RAM: Total 32121 MB ; Available 8826 MB
00:00:00.206 |N| Application PrintSystemInfo: Logs Enabled: Error, Guest
00:00:00.207 |N| Application PrintSystemInfo: Launch Mode: UserProfile
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
00:00:00.381 |N| Application LoadApplication: Using Firmware Version: 14.0.0
00:00:01.798 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA Corporation NVIDIA GeForce GTX 980 Ti/PCIe/SSE2 (4.6.0 NVIDIA 510.60.02)
'c != NULL' failed at ../pipewire-jack/src/pipewire-jack.c:3377 jack_client_close()
00:09:54.756 |G| HLE.OsThread.16 ServiceLm Log: Guest Log:
  Log level: Info
    Time: 594s
    Line: 18
    Filename: 'D:/home/Cafe/U-King/trunk/Game/Lib/sead/engine/library/modules/src/basis/nin/seadRawPrintNin.cpp'
    Function: 'PrintStringImpl'
    Module: ''
    Thread: 'MainThread'
    ProgramName: 'U-King'
    Message: '0000002e'

00:39:47.107 |E| HLE.GuestThread.22 Application : Unhandled exception caught: System.InvalidOperationException: Decoded a single empty exit block. Entry point = 0x1600001705.
   at ARMeilleure.Decoders.Decoder.Decode(IMemoryManager memory, UInt64 address, ExecutionMode mode, Boolean highCq, DecoderMode dMode) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Decoders\Decoder.cs:line 149
   at ARMeilleure.Translation.Translator.Translate(UInt64 address, ExecutionMode mode, Boolean highCq, Boolean singleStep) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 258
   at ARMeilleure.Translation.Translator.GetOrTranslate(UInt64 address, ExecutionMode mode) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 227
   at ARMeilleure.Instructions.NativeInterface.GetFunctionAddress(UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Instructions\NativeInterface.cs:line 240
Unhandled exception. System.InvalidOperationException: Decoded a single empty exit block. Entry point = 0x1600001705.
   at ARMeilleure.Decoders.Decoder.Decode(IMemoryManager memory, UInt64 address, ExecutionMode mode, Boolean highCq, DecoderMode dMode) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Decoders\Decoder.cs:line 149
   at ARMeilleure.Translation.Translator.Translate(UInt64 address, ExecutionMode mode, Boolean highCq, Boolean singleStep) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 258
   at ARMeilleure.Translation.Translator.GetOrTranslate(UInt64 address, ExecutionMode mode) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 227
   at ARMeilleure.Instructions.NativeInterface.GetFunctionAddress(UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Instructions\NativeInterface.cs:line 240
[2]    15219 IOT instruction  ./Ryujinx
```

**Comment by MutantAura** [comment]
#### Game Update Version : 1.6.0

#### Current on `master` : 1.1.248

#### Graphics Backend : Vulkan & OpenGL

Game renders and performs reasonably well on both backends with some minor graphical issues. Game will randomly crash with a GPU memory related error after an inconsistent amount of time.

#### Hardware Specs : 

##### CPU: Ryzen 5 5600X
##### GPU: NVIDIA RTX 3070
##### RAM: 32GB
##### OS: Windows 11 21H2

#### Known issues:

1. https://github.com/Ryujinx/Ryujinx/issues/3213
2. https://github.com/Ryujinx/Ryujinx/issues/2314
3. Terrain textures will drastically chain colours after using the master cycle rune. These effects can happen instantly or after some delay, warping to a new location or restarting will fix this.
![image](https://user-images.githubusercontent.com/44103205/188268482-76b642e6-e958-4fff-b11f-137a3e757802.png)

#### Screenshots
![image](https://user-images.githubusercontent.com/44103205/188268978-2b6762a8-647e-43f2-bb13-2c46d20e54b4.png)
![image](https://user-images.githubusercontent.com/44103205/188268515-a19c017f-ff62-4ac4-a493-9b2f38a78873.png)
![image](https://user-images.githubusercontent.com/44103205/188268594-5c8e5642-ac80-4b2e-acee-e6c7d19e73ee.png)
![image](https://user-images.githubusercontent.com/44103205/188268635-15ade9aa-063b-427a-b826-012eec1c1c95.png)
![image](https://user-images.githubusercontent.com/44103205/188268641-28ca03b5-73f9-42c1-8f8c-f28fb9ab6ab7.png)
![image](https://user-images.githubusercontent.com/44103205/188268692-9d13fa76-2da6-40c8-87ea-38351a392d14.png)
![image](https://user-images.githubusercontent.com/44103205/188268793-0c8bac42-7f57-42f5-80f3-154c2fc55857.png)
![image](https://user-images.githubusercontent.com/44103205/188268847-4836580b-aebb-43df-a381-52949f641138.png)
![image](https://user-images.githubusercontent.com/44103205/188268912-5dfff68b-f0cd-4efe-b63e-bac87ff8dd67.png)
![image](https://user-images.githubusercontent.com/44103205/188268957-4849e90d-d300-439e-a423-050cc853769c.png)

**Comment by MutantAura** [comment]
Updated

**Comment by MutantAura** [comment]
Updated.

**Comment by MelonSpeedruns** [comment]
The game plays wonderfully with and doesn't crash anymore randomly since the 1.1.295 version of Ryujinx.

Unfortunately, I am encountering a semi-consistent crash that seems to only occur when you're fighting the Lynel when doing the quest for the Shock Arrows near Zora's Domain. The crash seems to happen after a few minutes when staying in the area.

I have provided my save file as an attachment:

[Crash Save.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/9742616/Crash.Save.zip)

**Comment by fraz0815** [comment]
1.1.299 breaks vulkan on linux for me. Menu works, crashes during loading of a savegame

System:
```
0:00:00.422 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.299
00:00:00.433 |N| Application Print: Operating System: Linux 6.0.0-1-MANJARO #1 SMP PREEMPT_DYNAMIC Mon Oct 3 21:42:15 CEST 2022 (X64)
00:00:00.433 |N| Application Print: CPU: AMD Ryzen 7 5800X3D 8-Core Processor ; 16 logical
00:00:00.434 |N| Application Print: RAM: Total 32019 MB ; Available 27688 MB
00:00:00.436 |N| Application PrintSystemInfo: Logs Enabled: Error
00:00:00.437 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:00.432 |N| Application LoadApplication: Using Firmware Version: 11.0.0
00:00:02.711 |N| GUI.RenderLoop Gpu PrintGpuInformation: AMD AMD RADV NAVI10 (Vulkan v1.3.211, Driver v22.1.7)
```

Last lines of log
```
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First dstAccessMask is VK_ACCESS_SHADER_READ_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second dstAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First srcAccessMask is VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second srcAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First dstAccessMask is VK_ACCESS_SHADER_READ_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second dstAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First srcAccessMask is VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second srcAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First dstAccessMask is VK_ACCESS_SHADER_READ_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second dstAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First srcAccessMask is VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second srcAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdDrawIndexed-renderPass-02684 ] Object 0: handle = 0x85bd19000000f072, type = VK_OBJECT_TYPE_RENDER_PASS; Object 1: handle = 0x40b0940000000489, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x8cb637c2 | vkCmdDrawIndexed: RenderPasses incompatible between active render pass w/ VkRenderPass 0x85bd19000000f072[] and pipeline state object w/ VkRenderPass 0x40b0940000000489[]: First dstAccessMask is VK_ACCESS_SHADER_READ_BIT|VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT, but second dstAccessMask is VK_ACCESS_MEMORY_READ_BIT|VK_ACCESS_MEMORY_WRITE_BIT. The Vulkan spec states: The current render pass must be compatible with the renderPass member of the VkGraphicsPipelineCreateInfo structure specified when creating the VkPipeline bound to VK_PIPELINE_BIND_POINT_GRAPHICS (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdDrawIndexed-renderPass-02684)
00:00:24.964 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdBindVertexBuffers2-pOffsets-03357 ] Object 0: handle = 0x2cfba2000000001c, type = VK_OBJECT_TYPE_BUFFER; | MessageID = 0x206e8786 | vkCmdBindVertexBuffers2EXT offset (0x10) is beyond the end of the buffer. The Vulkan spec states: All elements of pOffsets must be less than the size of the corresponding element in pBuffers (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdBindVertexBuffers2-pOffsets-03357)
00:00:24.965 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-vkCmdBindVertexBuffers2-pOffsets-03357 ] Object 0: handle = 0x2cfba2000000001c, type = VK_OBJECT_TYPE_BUFFER; | MessageID = 0x206e8786 | vkCmdBindVertexBuffers2EXT offset (0x10) is beyond the end of the buffer. The Vulkan spec states: All elements of pOffsets must be less than the size of the corresponding element in pBuffers (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkCmdBindVertexBuffers2-pOffsets-03357)
00:00:24.966 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-VkSamplerCreateInfo-customBorderColors-04085 ] Object 0: handle = 0x7dfa247d64b0, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x5e78a93b | vkCreateSampler(): A custom border color was specified without enabling the custom border color feature The Vulkan spec states: If the customBorderColors feature is not enabled, borderColor must not be VK_BORDER_COLOR_FLOAT_CUSTOM_EXT or VK_BORDER_COLOR_INT_CUSTOM_EXT (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkSamplerCreateInfo-customBorderColors-04085)
00:00:24.966 |E| GUI.RenderLoop Gpu DebugReport: Validation Error: [ VUID-VkSamplerCustomBorderColorCreateInfoEXT-format-04014 ] Object 0: handle = 0x7dfa247d64b0, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0xaa7bd161 | vkCreateSampler(): A custom border color was specified as VK_FORMAT_UNDEFINED without the customBorderColorWithoutFormat feature being enabled The Vulkan spec states: If the customBorderColorWithoutFormat feature is not enabled then format must not be VK_FORMAT_UNDEFINED (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkSamplerCustomBorderColorCreateInfoEXT-format-04014)
```

**Comment by olvlvl** [comment]
Hi everyone,

Ever since I visited the Akkala region (top right corner) the game colors have been glitching, even in zones that were perfectly fine before. Teleporting is not fixing it. I've tested with Vulkan and OpenGL. I'm playing on Windows 11.

Here's a video: https://drive.google.com/file/d/1O-StboUHUAhfSryI0WEPMKuTkr_yqvcr/view?usp=sharing

**Comment by MelonSpeedruns** [comment]
Got another crash while playing through the game, this one seems to be graphics related? 

[Ryujinx_1.1.298_2022-10-13_11-21-59.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/9778425/Ryujinx_1.1.298_2022-10-13_11-21-59.log)

**Comment by Grimish-ng** [comment]
Got the rainbow cycling... Gpu is winning, wink wink nudge nudge. That about covers it on the net7.

**Comment by R-033** [comment]
This is the result of taking a photo with camera rune on macOS (MoltenVK):
![Screenshot 2022-11-28 at 02 32 43](https://user-images.githubusercontent.com/15189406/204148756-8ad3a5e2-0b0f-47c5-a218-574c9b9fd6c0.png)

**Comment by Grimish-ng** [comment]
Hey, I'm surprised you got a picture with MoltenVK! My hats off to you.

**Comment by MutantAura** [comment]
> i'm on mac M1 Max Monterey and can't seem to get past the beginning beige screen after the initial flare and zelda's muffled voice. it just seems to freeze/hang. i'm using 1.6.0 update. any setting i should be aware of? BTW other games seem to run fine like Persona 5 Royal.

BoTW is not stable under the Hypervisor at the moment. You can untick the 'use hypervisor' setting and use the ARM64 JIT but performance will be heavily reduced.

**Comment by Jerry-XU1010** [comment]
> > > i'm on mac M1 Max Monterey and can't seem to get past the beginning beige screen after the initial flare and zelda's muffled voice. it just seems to freeze/hang. i'm using 1.6.0 update. any setting i should be aware of? BTW other games seem to run fine like Persona 5 Royal.
> > 
> > 
> > BoTW is not stable under the Hypervisor at the moment. You can untick the 'use hypervisor' setting and use the ARM64 JIT but performance will be heavily reduced.
> 
> I am on M1 pro. Tried JIT but the fps stays at 20, barely playable. But previous post on x64 seems to reach 30fps. Is it because m1 pro is too weak or ARM64 JIT can be improved

I am on M1 Max, but the FPS stays at 5 or 6

**Comment by BismuthUupa** [comment]
> The game plays wonderfully and doesn't crash anymore randomly since the 1.1.295 version of Ryujinx.
> 
> Unfortunately, I am encountering a semi-consistent crash that seems to only occur when you're fighting the Lynel when doing the quest for the Shock Arrows near Zora's Domain. The crash seems to happen after a few minutes when staying in the area.
> 
> This was my log when encountering the crash:
> 
> ```
> 00:00:00.055 |I| Configuration LogValueChange: ResScaleCustom set to: 1
> 00:00:00.055 |I| Configuration LogValueChange: MaxAnisotropy set to: 16
> 00:00:00.055 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
> 00:00:00.056 |I| Configuration LogValueChange: BackendThreading set to: Auto
> 00:00:00.057 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
> 00:00:00.058 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x2182
> 00:00:00.072 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.298
> 00:00:00.077 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
> 00:00:00.077 |N| Application Print: CPU: AMD Ryzen 5 5600X 6-Core Processor ; 12 logical
> 00:00:00.077 |N| Application Print: RAM: Total 32689 MB ; Available 24182 MB
> 00:00:00.080 |N| Application PrintSystemInfo: Logs Enabled: Error
> 00:00:00.080 |N| Application PrintSystemInfo: Launch Mode: Custom Path D:\Ryujinx Data
> 00:00:00.264 |N| Application LoadApplication: Using Firmware Version: 13.2.0
> 00:00:02.249 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA Corporation NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 (4.6.0 NVIDIA 516.94)
> 00:05:05.712 |E| HLE.GuestThread.37 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x00000021CE8D4000.
> 00:05:05.875 |E| HLE.GuestThread.37 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
>    at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise) in D:\a\Ryujinx\Ryujinx\Ryujinx.Memory\Tracking\MemoryTracking.cs:line 251
>    at ARMeilleure.Translation.Translator.Execute(ExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 124
>    at Ryujinx.Cpu.Jit.JitCpuContext.Execute(IExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\Ryujinx.Cpu\Jit\JitCpuContext.cs:line 32
>    at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\ArmProcessContext.cs:line 40
>    at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\Kernel\Threading\KThread.cs:line 1265
> ```
> 
> And here is my save file as an attachment:
> 
> [Crash Save.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/9742616/Crash.Save.zip)

Precisely same issue, got through it somehow violently saving every few seconds of progress, took me around 20-30 tries. It helped NOT going up the hill.

I found another instance of this in [Kah Mael Shrine](https://www.zeldadungeon.net/wiki/Kah%20Mael%20Shrine) / Drop and Rise on Tingel Island, which is pretty easily reproducable by arrowing down the box in the upper right and using it the intended way. I usually get the crash when a magnesis object drops on something moveable, which is quite rare but this shrine is like its own micro-dimension of this crash.

**Comment by EmulationFanatic** [comment]
For tracking purposes
Bug: bike nuke

**Comment by jradut** [comment]
As soon as I load up the game and move around the screen goes white except for the map and icons. I can open the menu and see it fine. Also sometime the colors go green, yellow then white.

BOTW 1.60
Ryujinx 1.1.468
Mods - Cheats
![Screenshot 2022-12-12 171016](https://user-images.githubusercontent.com/24232393/207206904-dcaaaa38-4c1e-460b-be07-1a30a316b04d.png)

**Comment by jradut** [comment]
Also get this error when i try to load it up. 

00:00:13.422 |E| HLE.GuestThread.15 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x00000000000F4000.
00:00:13.461 |E| HLE.GuestThread.15 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
   at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise) in D:\a\Ryujinx\Ryujinx\Ryujinx.Memory\Tracking\MemoryTracking.cs:line 271
   at ARMeilleure.Translation.Translator.Execute(ExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 130
   at Ryujinx.Cpu.Jit.JitCpuContext.Execute(IExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\Ryujinx.Cpu\Jit\JitCpuContext.cs:line 32
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\ArmProcessContext.cs:line 40
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\Kernel\Threading\KThread.cs:line 1265
   at System.Threading.Thread.StartCallback()

**Comment by MelonSpeedruns** [comment]
> As soon as I load up the game and move around the screen goes white except for the map and icons. I can open the menu and see it fine. Also sometime the colors go green, yellow then white.
> 
> BOTW 1.60 Ryujinx 1.1.468 Mods - Cheats ![Screenshot 2022-12-12 171016](https://user-images.githubusercontent.com/24232393/207206904-dcaaaa38-4c1e-460b-be07-1a30a316b04d.png)
> 
> 00:06:08.720 |I| Hid Configure: Configured Controller ProController to Player1 00:06:08.720 |I| GUI.WindowThread Hid SetupNpad: Connected Controller ProController to Player1 00:06:08.860 |S| HLE.OsThread.16 ServiceHid Start: Stubbed. ControllerApplet ReturnResult 1 0 00:06:08.869 |S| HLE.OsThread.16 ServiceAm SetIdleTimeDetectionExtension: Stubbed. {idleTimeDetectionExtension: 0} 00:06:08.870 |S| HLE.OsThread.9 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075} 00:06:08.879 |S| HLE.OsThread.9 ServiceHid SetGyroscopeZeroDriftMode: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075 ; _gyroscopeZeroDriftMode: Tight} 00:06:15.311 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 304161 bytes, profiled functions: 88069). 00:06:16.933 |I| HLE.OsThread.16 ServicePrepo ProcessReport: PlayReport log: UserId: 00000000000000010000000000000000 Room: option Report: { "BalloonTextOnOff": 1, "AutoSaveOnOff": 1, "CameraUpDownReverse": 0, "CameraRLReverse": 0, "TextRubyOnOff": 1, "JumpButtonChange": 0, "StickSensitivity": 2, "GyroOnOff": 1, "MiniMapDirection": 0, "AmiiboItemOnOff": 1, "MainScreenOnOff": 0, "DisplaySpecialMode": 0, "TextWindowChange": 1, "DockInOnOff": 1, "ControllerType": 1, "AttachExtGrip": 0, "CtrlModeHandheldFree": 0, "CtrlModeHandheldExt": 0, "CtrlModeHandheldFullKey": 1483, "CtrlModeHandheldHandheld": 0, "CtrlModeConsoleFree": 0, "CtrlModeConsoleExt": 0, "CtrlModeConsoleFull": 36665, "PlayTimeHandheld": 1483, "PlayTimeConsole": 36670, "PlayTimeAll": 38153, "RomVer": 160, "AoCVer": 0, "RegionLanguage": 16, "AudioMono": 0, "AudioStereo": 0, "Audio5_1ch": 24885, "AudioOther": 0 }
> 
> 00:06:26.192 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU... 00:06:31.145 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU... 00:06:45.310 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 304158 bytes, profiled functions: 88069). 00:07:15.310 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 304270 bytes, profiled functions: 88101).

The issue is due to you having the yellow stamina bar afaik. Try emptying it?

**Comment by mjosephabie** [comment]
> > > The game plays wonderfully and doesn't crash anymore randomly since the 1.1.295 version of Ryujinx.
> > > Unfortunately, I am encountering a semi-consistent crash that seems to only occur when you're fighting the Lynel when doing the quest for the Shock Arrows near Zora's Domain. The crash seems to happen after a few minutes when staying in the area.
> > > This was my log when encountering the crash:
> > > ```
> > > 00:00:00.055 |I| Configuration LogValueChange: ResScaleCustom set to: 1
> > > 00:00:00.055 |I| Configuration LogValueChange: MaxAnisotropy set to: 16
> > > 00:00:00.055 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
> > > 00:00:00.056 |I| Configuration LogValueChange: BackendThreading set to: Auto
> > > 00:00:00.057 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
> > > 00:00:00.058 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x2182
> > > 00:00:00.072 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.298
> > > 00:00:00.077 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
> > > 00:00:00.077 |N| Application Print: CPU: AMD Ryzen 5 5600X 6-Core Processor ; 12 logical
> > > 00:00:00.077 |N| Application Print: RAM: Total 32689 MB ; Available 24182 MB
> > > 00:00:00.080 |N| Application PrintSystemInfo: Logs Enabled: Error
> > > 00:00:00.080 |N| Application PrintSystemInfo: Launch Mode: Custom Path D:\Ryujinx Data
> > > 00:00:00.264 |N| Application LoadApplication: Using Firmware Version: 13.2.0
> > > 00:00:02.249 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA Corporation NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 (4.6.0 NVIDIA 516.94)
> > > 00:05:05.712 |E| HLE.GuestThread.37 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x00000021CE8D4000.
> > > 00:05:05.875 |E| HLE.GuestThread.37 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
> > >    at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise) in D:\a\Ryujinx\Ryujinx\Ryujinx.Memory\Tracking\MemoryTracking.cs:line 251
> > >    at ARMeilleure.Translation.Translator.Execute(ExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 124
> > >    at Ryujinx.Cpu.Jit.JitCpuContext.Execute(IExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\Ryujinx.Cpu\Jit\JitCpuContext.cs:line 32
> > >    at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\ArmProcessContext.cs:line 40
> > >    at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\Kernel\Threading\KThread.cs:line 1265
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > And here is my save file as an attachment:
> > > [Crash Save.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/9742616/Crash.Save.zip)
> > 
> > 
> > Precisely same issue, got through it somehow violently saving every few seconds of progress, took me around 20-30 tries. It helped NOT going up the hill.
> > I found another instance of this in [Kah Mael Shrine](https://www.zeldadungeon.net/wiki/Kah%20Mael%20Shrine) / Drop and Rise on Tingel Island, which is pretty easily reproducable by arrowing down the box in the upper right and using it the intended way. I usually get the crash when a magnesis object drops on something moveable, which is quite rare but this shrine is like its own micro-dimension of this crash.
> 
> There is two other shrines like this, the first one you can circumvent by paying close attention to dropping cubes or chests STRAIGHT onto the scales. No chance of that working in Yah Rin since you need to cross double-scales, you get quite far though if you drop them straight.
> 
> I also noticed I encounter the Lynel-version of this crash only in Rain and when one spots me. Hope it helps somehow, otherwise the game runs butter smooth on a GTX 1070 and an i7, and it's not like you can get stuck anyways (I just hope Hyrule castle has no scale puzzles)

I got the same error. I noticed it's always crash when fighting Lynel, especially if you fight him heads on no matter the weather (Rain, Clear, Snow). Unless it got fixed, better stay away from Master Sword Final Trials

**Comment by yogithesymbian** [comment]
anyone else have tried on mbp m1 for this game ? please confirm or refer a links that show off/preview.
btw i have tried pokemon legend arceus and its work like a charm.

**Comment by TechnikEmpire** [comment]
> I got the same error. I noticed it's always crash when fighting Lynel, especially if you fight him heads on no matter the weather (Rain, Clear, Snow). Unless it got fixed, better stay away from Master Sword Final Trials

Should probably split into a new issue.

There are two ways to make the color go crazy that I know of: 
1) use the master cycle
2) sleep in down rito bed at rito village where it gives you an extra stamina wheel. You have to deplete the extra stamina wheel in order to stop the color flickering. Same idea as master cycle, once you make it despawn, the effect goes away.

**Comment by anusmith** [comment]
Crashes and/or gets stuck after cutscene is over on M1 Mac. Disabling PPTC and Hypervisor fixes the problem.

[Ryujinx_ 1.1.0-macos1_2023-01-17_18-24-58.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/10435226/Ryujinx_.1.1.0-macos1_2023-01-17_18-24-58.log)

**Comment by leachenlol** [comment]
how can i find the download link??

**Comment by MetrosexualGarbodor** [comment]
Needs an update.

**Comment by pearabowls** [comment]
> > > > The game plays wonderfully and doesn't crash anymore randomly since the 1.1.295 version of Ryujinx.
> > > > Unfortunately, I am encountering a semi-consistent crash that seems to only occur when you're fighting the Lynel when doing the quest for the Shock Arrows near Zora's Domain. The crash seems to happen after a few minutes when staying in the area.
> > > > This was my log when encountering the crash:
> > > > ```
> > > > 00:00:00.055 |I| Configuration LogValueChange: ResScaleCustom set to: 1
> > > > 00:00:00.055 |I| Configuration LogValueChange: MaxAnisotropy set to: 16
> > > > 00:00:00.055 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
> > > > 00:00:00.056 |I| Configuration LogValueChange: BackendThreading set to: Auto
> > > > 00:00:00.057 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
> > > > 00:00:00.058 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x2182
> > > > 00:00:00.072 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.298
> > > > 00:00:00.077 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
> > > > 00:00:00.077 |N| Application Print: CPU: AMD Ryzen 5 5600X 6-Core Processor ; 12 logical
> > > > 00:00:00.077 |N| Application Print: RAM: Total 32689 MB ; Available 24182 MB
> > > > 00:00:00.080 |N| Application PrintSystemInfo: Logs Enabled: Error
> > > > 00:00:00.080 |N| Application PrintSystemInfo: Launch Mode: Custom Path D:\Ryujinx Data
> > > > 00:00:00.264 |N| Application LoadApplication: Using Firmware Version: 13.2.0
> > > > 00:00:02.249 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA Corporation NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 (4.6.0 NVIDIA 516.94)
> > > > 00:05:05.712 |E| HLE.GuestThread.37 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x00000021CE8D4000.
> > > > 00:05:05.875 |E| HLE.GuestThread.37 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
> > > >    at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise) in D:\a\Ryujinx\Ryujinx\Ryujinx.Memory\Tracking\MemoryTracking.cs:line 251
> > > >    at ARMeilleure.Translation.Translator.Execute(ExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\ARMeilleure\Translation\Translator.cs:line 124
> > > >    at Ryujinx.Cpu.Jit.JitCpuContext.Execute(IExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\Ryujinx.Cpu\Jit\JitCpuContext.cs:line 32
> > > >    at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\ArmProcessContext.cs:line 40
> > > >    at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in D:\a\Ryujinx\Ryujinx\Ryujinx.HLE\HOS\Kernel\Threading\KThread.cs:line 1265
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > And here is my save file as an attachment:
> > > > [Crash Save.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/9742616/Crash.Save.zip)
> > > 
> > > 
> > > Precisely same issue, got through it somehow violently saving every few seconds of progress, took me around 20-30 tries. It helped NOT going up the hill.
> > > I found another instance of this in [Kah Mael Shrine](https://www.zeldadungeon.net/wiki/Kah%20Mael%20Shrine) / Drop and Rise on Tingel Island, which is pretty easily reproducable by arrowing down the box in the upper right and using it the intended way. I usually get the crash when a magnesis object drops on something moveable, which is quite rare but this shrine is like its own micro-dimension of this crash.
> > 
> > 
> > There is two other shrines like this, the first one you can circumvent by paying close attention to dropping cubes or chests STRAIGHT onto the scales. No chance of that working in Yah Rin since you need to cross double-scales, you get quite far though if you drop them straight.
> > I also noticed I encounter the Lynel-version of this crash only in Rain and when one spots me. Hope it helps somehow, otherwise the game runs butter smooth on a GTX 1070 and an i7, and it's not like you can get stuck anyways (I just hope Hyrule castle has no scale puzzles)
> 
> I got the same error. I noticed it's always crash when fighting Lynel, especially if you fight him heads on no matter the weather (Rain, Clear, Snow). Unless it got fixed, better stay away from Master Sword Final Trials

> I usually crash when changing FOVs during the fight with the Lynel.  I crash majority of the time when locked in with the Lynel and after flurry rushes. I had no problem traversing the land close to the Lynel but only crashed when in combat.  This crash happens all around the map for me.

**Comment by TechnikEmpire** [comment]
> Needs an update. Bike nuke should be fixed.

In latest release bike is fixed for me. Also I've not had a crash fighting any Lynel. Have fought three, two at Hyrule Castle and one in gerudo highlands. However I did also recently clear and rebuild shader cache.

Should have at least left relevant specs before:
Win 10 x64
Ryzen 7 5800x
RTX 3070 Ti with Vulkan backend

**Comment by pouetiz** [comment]
Hello,

I experience crashes using Vulkan or OpenGL

Context:
Using the scales in Yah Rin Shrine

Spec:
Ryujinx 1.1.646
Win 11 x64
Ryzen 7 5700G
RTX 2060

Crash with Vulkan backend
[Ryujinx_1.1.646_2023-03-02_17-26-51.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/10875031/Ryujinx_1.1.646_2023-03-02_17-26-51.log)

Crash with OpenGL backend
[Ryujinx_1.1.646_2023-03-02_17-29-38.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/10875030/Ryujinx_1.1.646_2023-03-02_17-29-38.log)

**Comment by MutantAura** [comment]
Seems there is some corruption on the Great Plateau prior to the tower popping out of the ground: 

https://user-images.githubusercontent.com/44103205/232619266-d0df82d9-374a-4193-99e7-ca87d9b32a72.mp4

Save file directly at the spot is attached. 
RTX 3070 (531.something). Could be driver related.
[0.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11256120/0.zip)

**Comment by MelonSpeedruns** [comment]
> Seems there is some corruption on the Great Plateau prior to the tower popping out of the ground:
> 
>  Ryujinx.Nvidia.Profile.2023.04.17.-.22.44.25.04.Dv1r.mp4 
> Save file directly at the spot is attached. RTX 3070 (531.something). Could be driver related. [0.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11256120/0.zip)

Cemu has the same issue and can be fixed with the "Accurate Vulkan barriers" option. Does it happen on OpenGL too?

**Comment by OpaqueReptile** [comment]
Reproduced on Vulkan and OpenGL, on GTX 1080 531.41

https://user-images.githubusercontent.com/1264375/232626635-0db6631b-213b-4d59-a6ab-ab927da58024.mp4


https://user-images.githubusercontent.com/1264375/232626637-8d96263c-5aec-44a4-8f14-fb37a1791c5d.mp4

**Comment by OpaqueReptile** [comment]
> Seems there is some corruption on the Great Plateau prior to the tower popping out of the ground:
> 
>  Ryujinx.Nvidia.Profile.2023.04.17.-.22.44.25.04.Dv1r.mp4 
> Save file directly at the spot is attached. RTX 3070 (531.something). Could be driver related. [0.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11256120/0.zip)



> > Seems there is some corruption on the Great Plateau prior to the tower popping out of the ground:
> > Ryujinx.Nvidia.Profile.2023.04.17.-.22.44.25.04.Dv1r.mp4
> > Save file directly at the spot is attached. RTX 3070 (531.something). Could be driver related. [0.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11256120/0.zip)
> 
> Cemu has the same issue and can be fixed with their "Accurate Vulkan barriers" option. Does it happen on OpenGL too?

Seems to be fixed by https://github.com/Ryujinx/Ryujinx/pull/4601 in my testing

**Comment by buettnerj1** [comment]
similar issue, but not sure if it's the same issue as above (I think it probably is) multiple place in the castle.  Apologies if this is duplicate/not needed, but if help, save to same location
[botw.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11300084/botw.zip)

https://user-images.githubusercontent.com/68620279/233757873-af4df500-77c3-456a-b5f0-761ab1c7487e.mp4

**Comment by SpiritLetsPlays** [comment]
> similar issue, but not sure if it's the same issue as above (I think it probably is) multiple place in the castle. Apologies if this is duplicate/not needed, but if it helps, heres the save to same location as the video. Does it in both Vulkan and OpenGL. GPU: RTX2080
> 
> [botw.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/11300084/botw.zip)
> 
>  botw.flicker.mp4

What you're seeing is something called Z-Fighting, it's basically when 2 elements/textures are on the save level they try to overlap one another

**Comment by kiko245** [comment]
Dónde conseguir ROMs?

**Comment by nadimmoran** [comment]
Dump from you switch, not allowed shared roms game.

**Comment by GamerzHell9137** [comment]
### Game Name

The Legend of Zelda: Breath of the Wild

### Game Update Version

v1.6.0

### Emulator Build Version

1.1.804

### Graphics Backend

Vulkan

### Hardware Specs

- ##### CPU: *Ryzen 3600*
- ##### GPU: *NVIDIA RTX 3060 TI*
- ##### RAM: *16GB*


### Game State

Done a full playtrough with no crashes and performance was above of the full speed. There's only one blemish that shuldn't be happening and that's the tetris effect on various objects in the world. 

Above korok tree:

https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/6f71311d-91fb-4ba0-ae95-783c148abb6f

Gerudo town cooking place(Flickers in and out of existance) 

![ezgif-3-5c6bbcd80b](https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/a5b217cb-633a-4b67-8147-d9c1cde0a0b4)

Around the corners of the lake in Great Plateau :

https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/e5f258bc-f2cd-459e-899a-427962b7c2df


### Screenshots

![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/c10c10f1-e817-4ef3-9fe5-d23a11943f4a)
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/7c639bc5-e790-4525-aee8-b23d33c2b8c8)
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/2002038/48d64f16-11a0-4f7b-9a28-e58e816f81cf)



### Log File

_No response_

**Comment by crispyankles99** [comment]
I'm a noob how do I download this as a .zip?

**Comment by joroldan** [comment]
Glitchy screenshots in macOS version.

![Glitch in album previews](https://user-images.githubusercontent.com/10585314/257376451-0b36a059-89f0-4852-b7fd-968cf1fc814c.png)
![Glitch in images](https://user-images.githubusercontent.com/10585314/257376464-b6f29d6b-79ad-4b4c-820d-379f5569e4f7.png)

The error also happens in the encyclopedia screenshots and in the files stored in the user folder.

 Already reported in Ryujinx/Ryujinx#3917 and Ryujinx/Ryujinx#5511.

**Comment by XDMaleye** [comment]
Where do I download it?

**Comment by jcm93** [comment]
https://github.com/Ryujinx/Ryujinx-Games-List/assets/6864788/37dcaa81-0845-4407-bb44-e259a5b475df

macOS is playable in theory with the hypervisor now, but Magnesis appears unusable for some reason. Tested on shipped MoltenVK as well as a recently rebased fork to fix occlusion queries.

**Comment by cddude** [comment]
> > i'm on mac M1 Max Monterey and can't seem to get past the beginning beige screen after the initial flare and zelda's muffled voice. it just seems to freeze/hang. i'm using 1.6.0 update. any setting i should be aware of? BTW other games seem to run fine like Persona 5 Royal.
> 
> BoTW is not stable under the Hypervisor at the moment. You can untick the 'use hypervisor' setting and use the ARM64 JIT but performance will be heavily reduced.

When will this be fixed?  It's been unsupported since 2018.

**Comment by jcm93** [comment]
> When will this be fixed? It's been unsupported since 2018.

You can read https://github.com/Ryujinx/Ryujinx/pull/5898 and https://github.com/Ryujinx/Ryujinx/pull/6068 to get an idea of the issues at hand if you would like to take a look!

**Comment by lefthandfrypan** [comment]
> > > i'm on mac M1 Max Monterey and can't seem to get past the beginning beige screen after the initial flare and zelda's muffled voice. it just seems to freeze/hang. i'm using 1.6.0 update. any setting i should be aware of? BTW other games seem to run fine like Persona 5 Royal.
> > 
> > 
> > BoTW is not stable under the Hypervisor at the moment. You can untick the 'use hypervisor' setting and use the ARM64 JIT but performance will be heavily reduced.
> 
> When will this be fixed? It's been unsupported since 2018.

As @jcm93 has said; turning Hypervision off fixes the hang issue.

Settings > CPU and disable Hypervision. For the Memory Manager Mode: Host (fast)

I've put in 6 hours so far without many issues.

M1 MBP 2021, 16GB RAM

**Comment by SuperUnoCard** [comment]
how do i downloaod?

**Comment by ezamelczyk** [comment]
Will the hypervisor on macOS ever be fixed? Is that even something that's on a roadmap or is it just waiting for someone from the outside to come in and do it?

**Comment by helzes** [comment]
.

**Comment by loquito342** [comment]
does this got any progress?

**Comment by xXSerekXx** [comment]
hi, can somebody please help me? botw is crashing while i'm in divine beast.

**Comment by ekuland** [comment]
> hi, can somebody please help me? botw is crashing while i'm in divine beast.

[Join the discord for support.](https://discord.gg/ryujinx)
